src/ripple/app/misc/Validations.cpp

```diff
@@ -346,9 +347,12 @@ class ValidationsImp : public Validations
     }
 
     LedgerToValidationCounter getCurrentValidations (
-        uint256 currentLedger, uint256 priorLedger) override
+        uint256 currentLedger,
+        uint256 priorLedger,
+        LedgerIndex cutoffBefore) override
     {
         auto const cutoff = app_.timeKeeper().now().time_since_epoch().count() - LEDGER_VAL_INTERVAL;
+        auto const lCutoff = cutoff + LEDGER_VAL_INTERVAL - LEDGER_VAL_LOCAL;
```

Comment:
I think it's clearer if both `cutoff` and `lCutoff` were calculated from  a local `now` variable. I.e `cutoff = now -LEDGER_VAL_INTERVAL;`, `lCutoff = now - LEDGER_VAL_LOCAL;`